src/committee/node.rs

```diff
@@ -211,8 +211,8 @@ async fn round_2_signing(
     RpcResult::Ok(round2_response)
 }
 
-async fn is_alive(_params: Params<'static>, _context: Arc<NodeState>) -> String {
-    "hello".to_string()
+async fn is_alive(params: Params<'static>, _context: Arc<NodeState>) -> RpcResult<u64> {
+    Ok(params.parse::<[u64; 1]>()?[0].clone())
```

Comment:
Yup, a lot of APIs use the current timestamp in ping keepalive requests, so yeah it can be considered a standard. In this case it's more of a sanity check, but just in case the node has a weird caching reverse proxy set up, this is good to have.